### ISSUE TITLE
Skipping allele got error

### ISSUE THREAD

**Issue by adefelicibus** [griffithlab/pVAC-Seq#133]
Trying to run pipeline with example_data I got this error:
```
pvacseq run example_data/input.vcf Teste BoLA-T2C,BoLA-T2a,BoLA-T2b,BoLA-T2c,BoLA-T5,BoLA-T7 8 NetMHC /home/adefelicibus/pvac-seq/output/
Converting VCF to TSV
Completed
Generating Variant Peptide FASTA File
Completed
Splitting FASTA into smaller chunks - Chunk 1
Completed
Generating FASTA Key File - Chunk 1
Completed
Processing entries for Allele BoLA-T2C and Epitope Length 8 - Chunk 1
Running IEDB on Allele BoLA-T2C and Epitope Length 8 with Method NetMHC - Chunk 1
Completed
Parsing IEDB Output for Allele BoLA-T2C and Epitope Length 8 - Chunk 1
Completed
Processing entries for Allele BoLA-T2a and Epitope Length 8 - Chunk 1
Running IEDB on Allele BoLA-T2a and Epitope Length 8 with Method NetMHC - Chunk 1
Completed
Parsing IEDB Output for Allele BoLA-T2a and Epitope Length 8 - Chunk 1
Completed
Processing entries for Allele BoLA-T2b and Epitope Length 8 - Chunk 1
Running IEDB on Allele BoLA-T2b and Epitope Length 8 with Method NetMHC - Chunk 1
Completed
Parsing IEDB Output for Allele BoLA-T2b and Epitope Length 8 - Chunk 1
Completed
Processing entries for Allele BoLA-T2c and Epitope Length 8 - Chunk 1
Allele BoLA-T2c not valid for Method NetMHC. Skipping.
Parsing IEDB Output for Allele BoLA-T2c and Epitope Length 8 - Chunk 1
FileNotFoundError: [Errno 2] No such file or directory: '/home/adefelicibus/pvac-seq/output/Teste.BoLA-T2c.8.parsed.tsv1
```

Once the allele is skipped, it's not necessary to parse IEDB output, i guess.

**Comment by susannasiebert** [comment]
We just released pVAC-Seq 3.0.4 which should fix this issue.

**Comment by adefelicibus** [comment]
It's working. Thanks a lot!
